bench/00_transform/cub.cu

```diff
@@ -0,0 +1,80 @@
+#include "matx.h"
+#include <nvbench/nvbench.cuh>
+
+
+using namespace matx;
+
+//
+// types of data to run tests on
+//
+// using sort_types = nvbench::type_list<matxFp16Complex, cuda::std::complex<float>, cuda::std::complex<double>, float, double>;
+using sort_types = nvbench::type_list<double>;
+
+////////////////////////////////////////////////////////////////////////////////
+///
+///\brief benchmark for 1D Sorts
+///
+///\param param in
```

Comment:
The docstrings look incomplete. you can just remove them since these aren't parsed

Reply:
they are incomplete, I'll fill them out with meaningful data.

What do you mean "they're not parsed"? that's valid doxygen syntax, do you mean the benchmark functions are not parsed by doxy?

Reply:
we don't parse the benchmarks directly currently since they don't go anywhere

Reply:
ok cool. I went ahead and gave it meaningful content... the format pattern helps me with mental parsing so I'm ok with paying the dev maintenance to write the code.